rayon-core/src/lib.rs

```diff
@@ -207,6 +238,75 @@ impl ThreadPoolBuilder {
         registry.wait_until_primed();
         Ok(())
     }
+}
+
+impl ThreadPoolBuilder {
+    /// Create a scoped `ThreadPool` initialized using this configuration.
+    ///
+    /// This is a convenience function for building a pool using a `crossbeam`
+    /// scope to spawn threads in a [`spawn_handler`](#method.spawn_handler).
+    /// The threads in this pool will start by calling `wrapper`, which should
+    /// do initialization and continue by calling `ThreadBuilder::run()`.
```

Comment:
`build_scoped` and `spawn_handler` should probably have examples.

Reply:
Agreed -- the fact that they have to invoke run is pretty non-obvious, and best shown by example. (I know we say it but I think it'd be easily overlooked)